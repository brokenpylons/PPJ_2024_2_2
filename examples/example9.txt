var prod = 1;
for (var i = 0, 5) {
    print(prod);
    prod = 2 * prod;
}
