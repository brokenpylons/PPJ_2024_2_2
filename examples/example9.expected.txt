real("2") plus("+") real("2") divides("/") real("2") 
